packages/core/test/fst.test.ts

```diff
@@ -0,0 +1,13 @@
+import { fromReducer } from '../src/fst';
+
+describe('fst', () => {
```

Comment:
could u also add some tests for barebone fst interface? as well some example that would utilize `handleOutput`?